packages/commonwealth/client/scripts/views/Sublayout.tsx

```diff
@@ -41,6 +43,29 @@ const Sublayout = ({
     }
   }, []);
 
+  useEffect(() => {
+    console.log('wind', window.scrollY);
+    // Add a scroll event listener
+    const handleScroll = () => {
+      // Define the scroll point where you want to hide the div
+      const scrollPoint = 200; // Adjust this value as needed
+
+      // Check the current scroll position
+      if (window.scrollY >= scrollPoint) {
+        setIsVisible(true);
+      } else {
+        setIsVisible(false);
+      }
+    };
+
+    window.addEventListener('scroll', handleScroll);
+
+    return () => {
+      // Remove the scroll event listener
+      window.removeEventListener('scroll', handleScroll);
+    };
+  }, []);
```

Comment:
could you use some css to do that? You can use `position: sticky` or `position: fixed` for desktop resolution and using media query you can use static position in mobile resolution

Reply:
I misunderstood the AC's/Reqs. Spoke to product team and they cleared this up.

Reply:
1. long breadcrumbs should have horizontal scroll
![image](https://github.com/hicommonwealth/commonwealth/assets/14819225/6e88dac6-2c4c-490d-813c-3671190a78b7)

2. in general, breadcrumbs should not cover the body of the page. in several cases, breadcrumb covers the header
![image](https://github.com/hicommonwealth/commonwealth/assets/14819225/557ea8ca-f1e0-4b9a-af89-99d360739bde)